ocaml/lambda/lambda.ml

```diff
@@ -1596,7 +1598,10 @@ let primitive_result_layout (p : primitive) =
   | Pbigarrayref (_, _, kind, _) ->
       begin match kind with
       | Pbigarray_unknown -> layout_any_value
-      | Pbigarray_float32 | Pbigarray_float64 -> layout_boxed_float Pfloat64
+      | Pbigarray_float32 ->
+        (* CR mslater: (float32) bigarrays *)
+        layout_boxed_float Pfloat64
```

Comment:
Is `Pfloat64` correct here?

Reply:
Yeah - I just broke out the case so I could add the CR. Later on I will convert the float32 bigarray kind to actually give you back float32s.